### DISCUSSION TITLE
Reviewing entire notes based on their title ?

### DISCUSSION THREAD

**Discussion by clothilden** [st3v3nmw/obsidian-spaced-repetition#958] [Q&A]
Hi everyone, 
I've just discovered this plug-in and wanted to use it for something specific : Making flashcards with the title on one side and the content of the note on the other. 
So first is it possible ? 

If so, could someone help me cause I must have made some mistakes...

I have put the #review on my notes, and the plug-in seems to find them but only in the right panel... not in the flashcards window

I don't know if everything is clear but would really appreciate any help :)
Thank you!

<img width="1146" alt="Capture d’écran 2024-05-20 à 12 11 24" src="https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/170317944/2ed2a6c0-49eb-44f6-b813-feec339bfc2b">

**Comment by ronzulu** [comment]
Hi @clothilden within the plugin notes and flashcards are quite distinct concepts.

As you've noticed, adding `#review` specifies a file as being a note, but not a flashcard.

Flashcards are created with different tags, by default `#flashcards` https://www.stephenmwangi.com/obsidian-spaced-repetition/flashcards/

Sorry if this is already obvious.

What you are after is not a current feature. There is a suggestion along similar lines to yours at:
[ Entire note as one flashcard #729 ](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/729)

Cheers
Ronny
